h/streamer/messages.py

```diff
@@ -127,7 +127,7 @@ def handle_annotation_event(message, sockets, request, session):
 
     for socket in matching_sockets:
         # Don't send notifications back to the person who sent them
-        if message["src_client_id"] == socket.client_id:
+        if socket.client_id and message.get("src_client_id") == socket.client_id:
```

Comment:
This was needed at one point when the URL migration task was triggering realtime updates. This is no longer strictly required, but I think it makes sense to allow web-socket notifications to be dispatched without an associated `src_client_id` (this is an identifier for the Hypothesis client instance that triggered the notification by making an API call).

Reply:
This seems to be an unrelated change. Maybe make it in a separate PR, so you can deploy and test it separately and in case you need to revert it?

Reply:
OK, I'll take this change out of this PR.